sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/spanner/SpannerIO.java

```diff
@@ -329,12 +333,26 @@ public ReadAll withTimestampBound(TimestampBound timestampBound) {
       return toBuilder().setTimestampBound(timestampBound).build();
     }
 
+    /** If true the uses Cloud Spanner batch API. */
```

Comment:
Can you clarify in the documentation that batching is the default ? Alternatively, wow about just having a method withoutBatching() that can be used to disable batching ?

Reply:
Done. I think `without` method will be more confusing.